config.properties.example

```diff
@@ -76,9 +76,13 @@ password=maxwell
 #kafka.bootstrap.servers=hosta:9092,hostb:9092
 
 # kafka topic to write to
+# this can be static, e.g. 'maxwell', or dynamic, e.g. namespace_%{database}_%{table}
+# in the latter case 'database' and 'table' will be replaced with the values for the row being processed
 #kafka_topic=maxwell
 
 # kafka topic to write DDL to
+# this can be static, e.g. 'maxwell_ddl', or dynamic, e.g. namespace_ddl_%{database}_%{table}
+# in the latter case 'database' and 'table' will be replaced with the values for the row being processed
```

Comment:
this is not true, should actually include a note here about how ddl-topics are *not* dynamic.